packages/gestalt/src/TableOfContents.js

```diff
@@ -0,0 +1,44 @@
+// @flow strict
+import { type Node } from 'react';
+import Box from './Box.js';
+import Heading from './Heading.js';
+import styles from './TableOfContents.css';
+import TableOfContentsItemList from './TableOfContents/TableOfContentsItemList.js';
+import TableOfContentsItem from './TableOfContentsItem.js';
+
+type Props = {|
+  /**
+   * String that clients such as VoiceOver will read to describe the element.
+   */
+  accessibilityLabel?: string,
+  /**
+   * Title for the TableOfContents.
```

Comment:
can we add link to the variants? 
p.e.
See the [animation](https://deploy-preview-3118--gestalt.netlify.app/web/overlaypanel#Animation) variant to learn more.

Reply:
you mean for the `title` prop?